cluster/gce/util.sh

```diff
@@ -2895,6 +2895,59 @@ function config-partitionserver() {
 
 }
 
+
+##set apiserver datapartition config by apiserver total number
```

Comment:
I would prefer those logic to go to perf test. Environment set up should only have necessary information and do not care how the environment is used.

Reply:
Talk to Xiaoning if you think it is necessary to do it here.

Reply:
as my understanding, if we start 3 apiservers, we should have data partition ready for these 3 apiservers, doesn't mentor if we run perf-tests. data partition is part of multi apiserver, not part of perf-tests.

Reply:
I agree with Sonya. Configuration of the cluster is a setup thing, and not a test specific thing.

Reply:
Setup can also be multiple parts. API server partition is designed to be flexible, not tied to environments.